content/assignments/minishell/checker/_test/run_test.sh

```diff
@@ -0,0 +1,252 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+
+# ----------------- General declarations and util functions ------------------ #
+
+exec_name="mini-shell"
+ref_name="bash"
+ref_name_alt="dash"
+
+MAIN_TEST_DIR="_test/outputs"
+INPUT_DIR="_test/inputs"
+REFS_DIR="_test/refs"
+LOG_FILE="/dev/null"
+export max_points=95
+TEST_TIMEOUT=30
+
+TEST_LIB=_test/test_lib.sh
+export CHECK_SKIP='parser.\(\(tab\|yy\).\(c\|h\)\|h\)$'
+
+# load the lib functions
```

Comment:
```suggestion
# Load the lib functions.
```
Change all comments to start with capital letters and end with a dot (`.`).

Reply:
Done.